pages/index.html

```diff
@@ -73,32 +80,48 @@
 <!--  REST API-->
 
 <section id="rest-api" class="container-fluid">
-  <h2><span class="fa-cogs" aria-hidden="true"></span> Build apps with the REST API</h2>
-  <p>Create apps for our App Store or develop custom integrations using our REST API.</p>
-  <a href="/apps/" class="cta">Read REST API documentation</a>
+  <div class="row">
+    <div class="col-md-10  col-md-offset-1">
+      <h2><span class="fa-cogs" aria-hidden="true"></span> Build apps for ePages Online Shops with the REST API</h2>
+      <p>Create apps for our App Store or develop custom integrations using our REST API.</p>
+      <p>Sign up for free and join the ePages Developer Program. Gain access to all the API resources you need to develop and successfully publish your app.</p>
+      <p>Promote your service in the ePages App Store and earn up to 70% of the revenue from each sale to an <span style="white-space: nowrap">ePages merchant</span>.</p>
+      <a href="/apps/" class="cta">Read REST API documentation</a>
+    </div>
+  </div>
 </section>
 
 <!--  REGISTER-->
 
 <section id="register" class="container-fluid">
-  <h2>Not registered for the developer program yet?</h2>
-  <a data-toggle="modal" href="#modal-popup" class="cta">Join now</a>
+  <div class="row">
+    <div class="col-md-6">
+      <h2>Create Apps for ePages Now!</h2>
+      <p>ePages Now is our new product generation that let's you develop awesome storefront apps such as chat tools or exit popup plugins.</p>
+      <a data-toggle="modal" href="#modal-popup-now" class="cta">Join now</a>
+    </div>
+    <div class="col-md-6">
+      <h2>Create Apps for ePages 6!</h2>
+      <p>ePages 6 is our current full-blown shop solution that enables yout o get many different types of data for your service.</p>
+      <a data-toggle="modal" href="#modal-popup-6" class="cta">Join now</a>
+    </div>
+  </div>
 </section>
 
 <!--  MORE RESOURCES-->
 
 <section id="resources" class="container">
-  <h2>More resources</h2>
   <div class="row">
-    <div>
+    <h2>More resources</h2>
+    <div class="col-sm-8  col-sm-offset-2  col-md-4  col-md-offset-0">
       <h4><span aria-hidden="true" class="fa-cogs"></span>Developer Blog</h4>
-      <p>Read about the latest developments at ePages and current industry trends at <a href="{{ '/blog/index.html' | prepend: site.baseurl }}">our blog.</a></p>
+      <p>Read about the latest developments at ePages and current industry trends at <a href="{{ '/blog/index.html' | prepend: site.baseurl }}"><span style="white-space: nowrap">our blog</span>.</a></p>
     </div>
-    <div>
+    <div class="col-sm-8  col-sm-offset-2  col-md-4  col-md-offset-0">
       <h4><span aria-hidden="true" class="fa-pencil"></span>Open Source Projects</h4>
       <p>We have various open source projects which you can use and contribute to.</p>
```

Comment:
Please change to:

Our development teams benefit a lot from open-source software. Here's a list of our [open-source projects](https://github.com/ePages-de) that you can use and contribute to.